gstools/transform/field.py

```diff
@@ -335,12 +335,12 @@ def _zinnharvey(field, conn="high", mean=None, var=None):
         mean = np.mean(field)
     if var is None:
         var = np.var(field)
-    field = np.abs((field - mean) / np.sqrt(var))
```

Comment:
Why reverting this? I fixed just it in https://github.com/GeoStat-Framework/GSTools/commit/0f24aad69d2602436b8dbeceb48df33517b2356d

I think the std-deviation needs to used there